WMF Framework/AlertType.swift

```diff
@@ -0,0 +1,21 @@
+public enum AlertType {
```

Comment:
Do you think it would be ok to name this something like `ReadingListAlertType`? Just wondering about confusion with alerts with types elsewhere in the future...